pystructs/fields/integer.py

```diff
@@ -16,6 +18,13 @@ def __init__(self, size: int, byteorder='little'):
     def fetch(self) -> int:
         return int.from_bytes(super().fetch(), self.byteorder)
 
```

Comment:
This project follows code-style named `pep8`

Before defining functions, classes, you have to remain two blank lines